buildpack.md

```diff
@@ -894,7 +938,7 @@ If an `order` is specified, then `stacks` MUST NOT be specified.
 *Key: `api = "<buildpack API version>"`*
  - MUST be in form `<major>.<minor>` or `<major>`, where `<major>` is equivalent to `<major>.0`
  - MUST describe the implemented buildpack API.
- - SHOULD indicate the lowest compatible `<minor>` IF buildpack behavior is consistent with multiple `<minor>` versions of a given `<major>`
+ - SHOULD indicate the lowest compatible `<minor>` if buildpack behavior is consistent with multiple `<minor>` versions of a given `<major>`
```

Comment:
Is there a rhyme or reason to when we choose to bold these or not?

Reply:
That's a good question. I didn't bold this one b/c it wasn't bold before. I could reverse engineer the rational (not at the beginning of the sentence something something...) but I would just be making things up. Maybe @sclevine can articulate the rule here.